.github/workflows/test-template.yml

```diff
@@ -0,0 +1,1368 @@
+name: CI Template job
```

Comment:
I find the rename confusing as this file is not just a template, it's a workflow file (which has "on" triggers) but is also used as a template in the daily workflow files.

Reply:
Forget about this, I saw the test workflow file as deleted but it's actually still there :D

Reply:
The only `on` trigger it still has is `workflow_call`, which means it can only be called from other workflows. What would be a better name for it?

Reply:
(no need to do anything, sorry)